Beilagen/VegetarischeFrikadellen.md

```diff
@@ -0,0 +1,23 @@
+# Vegetarische Frikadellen
+- Gelingen leicht und sind schnell gemacht
+
+Die geschmackvolle Alternative zur Fleisch-Frikadelle, entweder als Beilage frisch gebraten oder zu einem knackigen Salat kalt servieren.
+
+
+## Zutaten
+* 150 g kernige Haferflocken
+* 100 g feine Haferflocken
+* 200 g Kräuterquark
+* 240 g geriebenen, würzigen Käse
+* 3 Eier
+* 1 Zwiebel
+* Salz
+* Pfeffer
+* Olivenöl
+
+## Anleitung
+1. Die Zwiebel in feine Stücke hacken und in Olivenöl glasig braten. Danach abkühlen lassen.
+2. Die übrigen Zutaten vermischen und die abgekühlten Zwiebeln dazu geben. Die Masse sollte nun klebrig sein.
+3. Mit Salz und Pfeffer abschmecken und für 30 Minuten in den Kühlschrank stellen.
+4. Nun kleine Frikadellen mit der Hand formen. (Bei mir werden es 20 bis 25 Stück.)
+5. Dann die Frikadellen in etwas Olivenöl auf niederer Stufe braten.
```

Comment:
Diese Datei sollte rausgenommen werden und in einem weiteren Pull Request submitted werden. Bitte einfach ohne die Datei neu commiten